test/specs/global.deprecations.tests.js

```diff
@@ -1,4 +1,80 @@
 describe('Deprecations', function() {
+	describe('Version 2.7.0', function() {
+		describe('Chart.Controller.update(duration, lazy)', function() {
+			var chart;
+			var addAnimationSpy;
+
+			beforeEach(function() {
+				chart = acquireChart({
```

Comment:
Those are declared inside the `describe`. But I can change them to use `this` if that's the usual pattern in the project.